lte/gateway/python/magma/mobilityd/subscriberdb_client.py

```diff
@@ -72,18 +75,18 @@ def get_subscriber_ip(self, sid: str) -> Optional[StaticIPInfo]:
         try:
             apn_config = self._find_ip_and_apn_config(sid)
             logging.debug("ip: Got APN: %s", apn_config)
-            if apn_config:
+            if apn_config and apn_config.assigned_static_ip:
                 return StaticIPInfo(ip=apn_config.assigned_static_ip,
                                     gw_ip=apn_config.resource.gateway_ip,
                                     gw_mac=apn_config.resource.gateway_mac,
                                     vlan=apn_config.resource.vlan_id)
 
-        except ValueError:
-            logging.warning("Invalid or missing data for sid %s: ", sid)
+        except ValueError as ex:
+            logging.warning("static Ip: Invalid or missing data for sid %s: : %s", sid, ex)
             raise SubscriberDBStaticIPValueError(sid)
```

Comment:
I think this error should be raised with the whole message instead of only the `sid`